Signal/src/ViewControllers/ShowGroupMembersViewController.m

```diff
@@ -250,8 +192,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                                                      [BlockListUIUtils
                                                          showUnblockPhoneNumberActionSheet:recipientId
```

Comment:
after blocking/unblocking the corresponding "Blocked"/"Unblocked" label doesn't change until you reload the view.